lib/nacs-seq/manager.cpp

```diff
@@ -391,6 +391,26 @@ void Manager::unref_data(uint64_t id)
     m_datainfo.erase(it);
 }
 
+NACS_EXPORT() uint32_t Manager::get_device_restart(const char *dname)
+{
+    std::string name(dname);
```

Comment:
Ah, meaning there'll be an implicit conversion to the std::string keytype when I use it in std::map::find?

Reply:
In the worst cast that should happen. The constructor used here isn’t explicit.

Reply:
I. The best case it should be able to avoid constructing the string but I’m not sure if the correct compare function is defined and used automatically.